packages/manager/modules/pci/src/projects/project/storages/volume-backup/volume-backup.routing.js

```diff
@@ -24,6 +24,27 @@ export default /* @ngInject */ ($stateProvider) => {
           'pci_projects_project_storages_volume_backup_breadcrumb',
         ),
 
+      guides: /* @ngInject */ (coreConfig, $translate) => (guides) => {
```

Comment:
**nitpick:** `guides` is used in two places.  Better to rename one of them to avoid confusion.

Reply:
done